apps/deepl_integration/serializers.py

```diff
@@ -63,17 +63,30 @@ class Status(models.IntegerChoices):
     status = serializers.ChoiceField(choices=Status.choices)
 
 
+class ImagePathSerializer(serializers.Serializer):
+    page_number = serializers.IntegerField(required=True)
+    images = serializers.ListField(
+        child=serializers.CharField(allow_blank=True),
+        default=[],
+    )
+
+
+class TablePathSerializer(serializers.Serializer):
+    page_number = serializers.IntegerField(required=True)
+    order = serializers.IntegerField(required=True)
+    image_link = serializers.URLField(required=True)
+    content_link = serializers.URLField(required=True)
+
+
 # -- Lead
 class LeadExtractCallbackSerializer(DeeplServerBaseCallbackSerializer):
     """
     Serialize deepl extractor
     """
     url = serializers.CharField(required=False)
     # Data fields
-    images_path = serializers.ListField(
-        child=serializers.CharField(allow_blank=True),
-        required=False, default=[],
-    )
+    images_path = serializers.ListSerializer(child=ImagePathSerializer(required=False))
+    tables_path = serializers.ListSerializer(child=TablePathSerializer(required=False))
```

Comment:
```suggestion
    images_path = serializers.ListSerializer(child=ImagePathSerializer(required=True), required=False)
    tables_path = serializers.ListSerializer(child=TablePathSerializer(required=True), required=False)
```

Also, let's check this behavior if user can send Null.